docs/docs/community/usage-tracking.mdx

```diff
@@ -1,17 +1,13 @@
----
-id: usage-tracking
----
-
 import { Table } from "@site/src/components/Table";
 
-# Telemetry
+# MLflow Telemetry
```

Comment:
This sounds like this is a feature for telemetry. Can we use `Usage Tracking` instead?

Reply:
Ok